zzk/service/service.go

```diff
@@ -65,6 +65,19 @@ type ServiceNode struct {
 	version interface{}
 }
 
+// Update implements zzk.Node
+func (node *ServiceNode) Update(conn client.Connection) error {
```

Comment:
I don't think this is used anywhere - can you remove it without breaking anything?